Gemfile

```diff
@@ -53,6 +53,7 @@ gem 'sshkey', '~> 1.9'
 gem 'dynflow', '>= 0.8.29', '< 1.0.0'
 gem 'daemons'
 gem 'get_process_mem'
+gem 'rdoc'
```

Comment:
not related / required?

Reply:
Foreman won't compile at my end without this line, you can ignore this because ill be working on my Mac soon and I wlil delete this.